composer.json

```diff
@@ -5,23 +5,11 @@
     "license": "GPL-2.0+",
     "require": {
         "php": "^7.1",
-        "knplabs/github-api": "^2.0",
-        "php-http/guzzle6-adapter": "^v1.1",
-        "michelf/php-markdown": "^1.7.0",
-        "erusev/parsedown": "^1.6.3",
-        "justinrainbow/json-schema": "^5.2",
-        "drupal/apigee_edge": "^1.0rc2",
-        "drupal/contact_emails": "^1.16",
         "drupal/core": "^8.7",
-        "drupal/eu_cookie_compliance": "^1.5",
-        "drupal/google_tag": "^1.1",
-        "drupal/honeypot": "^1.29",
-        "drupal/metatag": "^1.8",
-        "drupal/node_view_permissions": "^1.2",
-        "drupal/pathauto": "^1.4",
-        "drupal/redirect": "^1.2",
         "drupal/swagger_ui_formatter": "^2.3",
-        "drupal/views_accordion": "^1.2"
+        "drupal/views_accordion": "^1.2",
+        "erusev/parsedown": "^1.6.3",
```

Comment:
Minimum version mismatch, Guides requires at least 1.7. Should not we bump here as well?
https://github.com/Pronovix/devportal-drupal-module/blob/8.x-2.x/modules/guides/composer.json#L6

Reply:
You mean the `dp_faq` module? Because `guides` doesn't use `views_accordion` I guess.

Reply:
And where it is specified that `dp_faq` requires `1.7` minimum? I cannot find it.

Reply:
Other: why `guides` contains a `composer.json` file?

Reply:
Check the link above, it highlights the "erusev/parsedown" library.

Reply:
Ahh, so `erusev/parsedown` ok. I see. I was checking `views_accordion` for some reason :)